src/views/AccessControl/LocalUserManagement/LocalUserManagement.vue

```diff
@@ -1,10 +1,6 @@
 <template>
   <b-container class="ml-0">
-    <b-row>
-      <b-col lg="10">
-        <h1>Local user management</h1>
-      </b-col>
-    </b-row>
+    <PageTitle pageTitle="Local user management" />
```

Comment:
How do you feel about pulling this from the router meta data?  Also wondering if we need to be this verbose in our prop naming? You could bind the route meta.title...

`<PageTitle :pageTitle="this.$route.meta.title" />`

Reply:
Yeah, I think that's a good idea, I'll just remove the title prop all together and always pull from route meta